cmd/traffic/cmd/agent/agent.go

```diff
@@ -128,15 +132,9 @@ func Main(ctx context.Context, args ...string) error {
 	}
 	dlog.Infof(ctx, "%+v", config)
 
-	hostname, err := os.Hostname()
-	if err != nil {
-		dlog.Infof(ctx, "hostname: %+v", err)
-		hostname = fmt.Sprintf("unknown: %+v", err)
-	}
-
 	info := &rpc.AgentInfo{
```

Comment:
Is there a PR for changes to the proprietary agent for this change set?  I noticed when I ran the smoke tests, and tried to create an intercept with a preview URL I got this error:
```
telepresence: error: creating preview domain: rpc error: code = Unavailable desc = transport is closing
rpc error: code = Unavailable desc = connection closed
```

Similarly, I found this in the traffic-agent logs: 
```
time="2021-05-04 13:36:52" level=info msg="talking to manager:rpc error: code = Canceled desc = context canceled" func=main.sidecar file="github.com/datawire/telepresence2/proprietary/cmd/app-sidecar/sidecar.go:246" PID=1 THREAD=/sidecar
```
One of the arguments to TalkToManager is the AgentInfo so I suspect this change needs to be made in the proprietary agent repo as well.